lib/Runtime/Types/TypeId.h

```diff
@@ -2,10 +2,140 @@
+      TypeIds_UndeclBlockVar = 25,
+
+      TypeIds_LastStaticType = TypeIds_UndeclBlockVar,
+
+      TypeIds_Proxy = 26,
+      TypeIds_Function = 27,
+
+      //
+      // The backend expects only objects whose typeof() === "object" to have a
+      // TypeId >= TypeIds_Object. Only 'null' is a special case because it
+      // has a static type.
+      //
+      TypeIds_Object = 28,
+      TypeIds_Array = 29,
+      TypeIds_ArrayFirst = TypeIds_Array,
+      TypeIds_NativeIntArray = 30,
+    #if ENABLE_COPYONACCESS_ARRAY
+      TypeIds_CopyOnAccessNativeIntArray = 31,
+    #endif
+      TypeIds_NativeFloatArray = 32,
+      TypeIds_ArrayLast = TypeIds_NativeFloatArray,
+      TypeIds_ES5Array = 33,
+      TypeIds_ArrayLastWithES5 = TypeIds_ES5Array,
+      TypeIds_Date = 34,
+      TypeIds_RegEx = 35,
+      TypeIds_Error = 36,
+      TypeIds_BooleanObject = 37,
+      TypeIds_NumberObject = 38,
+      TypeIds_StringObject = 39,
+      TypeIds_BigIntObject = 40,
+      TypeIds_SIMDObject = 41,
+      TypeIds_Arguments = 42,
+      TypeIds_ArrayBuffer = 43,
+      TypeIds_Int8Array = 44,
+      TypeIds_TypedArrayMin = TypeIds_Int8Array,
+      TypeIds_TypedArraySCAMin = TypeIds_Int8Array, // Min SCA supported TypedArray TypeId
+      TypeIds_Uint8Array = 45,
+      TypeIds_Uint8ClampedArray = 46,
+      TypeIds_Int16Array = 47,
+      TypeIds_Uint16Array = 48,
+      TypeIds_Int32Array = 49,
+      TypeIds_Uint32Array = 50,
+      TypeIds_Float32Array = 51,
+      TypeIds_Float64Array = 52,
+      TypeIds_TypedArraySCAMax = TypeIds_Float64Array, // Max SCA supported TypedArray TypeId
+      TypeIds_Int64Array = 53,
+      TypeIds_Uint64Array = 54,
+      TypeIds_CharArray = 55,
+      TypeIds_BoolArray = 56,
+      TypeIds_TypedArrayMax = TypeIds_BoolArray,
+      TypeIds_EngineInterfaceObject = 57,
+      TypeIds_DataView = 58,
+      // 59 deleted
+      TypeIds_Map = 60,
+      TypeIds_Set = 61,
+      TypeIds_WeakMap = 62,
+      TypeIds_WeakSet = 63,
+      TypeIds_SymbolObject = 64,
+      TypeIds_ArrayIterator = 65,
+      TypeIds_MapIterator = 66,
+      TypeIds_SetIterator = 67,
+      TypeIds_StringIterator = 68,
+      // 69 - deleted
+      TypeIds_Generator = 70,
+      TypeIds_AsyncGenerator = 71,
+      TypeIds_AsyncFromSyncIterator = 72,
+      TypeIds_AwaitObject = 73,
+      TypeIds_Promise = 74,
+      TypeIds_SharedArrayBuffer = 75,
+
+      TypeIds_WebAssemblyModule = 76,
+      TypeIds_WebAssemblyInstance = 77,
+      TypeIds_WebAssemblyMemory = 78,
+      TypeIds_WebAssemblyTable = 79,
+
+      TypeIds_LastBuiltinDynamicObject = TypeIds_WebAssemblyTable,
+      TypeIds_GlobalObject = 80,
+      TypeIds_ModuleRoot = 81,
+      TypeIds_LastTrueJavascriptObjectType = TypeIds_ModuleRoot,
 
-#include "EdgeJavascriptTypeId.h"
-    // All WinRT dates are regular Javascript dates too
-    inline bool IsDateTypeId(TypeId typeId) { return (typeId == TypeIds_Date || typeId == TypeIds_WinRTDate); }
```

Comment:
Previously this file simply included "EdgeJavascriptTypeId.h" and defined IsDateTypeId which is now defunct - hence combined the files.